routes/teamAPI.js

```diff
@@ -8,10 +8,10 @@ var router = require('express').Router();
 //		name:String
 //	optional:
 //		description:String
-//		coach:String (user)
-//		lead:String(user)
-//		member:[String] (user)
-//		project: [String] (project)
+//		coach:id (user)
+//		lead:id(user)
```

Comment:
Is lead still here?  does it mean it is not required to have leader as the 1st member of members[]?